source/features/new-repo-disable-projects-and-wikis.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL} from '../github-helpers';
+
+async function disableWikiAndProjects(): Promise<void> {
+	sessionStorage.removeItem('rghNewRepo');
+
+	void api.v3(`repos/${getRepoURL()}`, {
+		method: 'PATCH',
+		body: {
+			has_projects: false,
+			has_wiki: false
+		}
+	});
+
+	const wiki = await elementReady('[data-tab-item="wiki-tab"]');
+	wiki?.closest('.d-flex')!.remove();
+	const projects = await elementReady('[data-content="Projects"]');
+	projects?.closest('.d-flex')!.remove();
+}
+
+function setStorage(): void {
+	sessionStorage.rghNewRepo = select<HTMLInputElement>('[name="rgh-disable-project"]')!.checked;
```

Comment:
Let’s not store it at all when false so you can avoid the already unnecessary `=== true` below